client_it_test.go

```diff
@@ -36,14 +36,55 @@ import (
 	"github.com/hazelcast/hazelcast-go-client/hzerrors"
 	"github.com/hazelcast/hazelcast-go-client/internal"
 	"github.com/hazelcast/hazelcast-go-client/internal/it"
-	"github.com/hazelcast/hazelcast-go-client/internal/proxy"
+	"github.com/hazelcast/hazelcast-go-client/internal/it/skip"
 	"github.com/hazelcast/hazelcast-go-client/logger"
 	"github.com/hazelcast/hazelcast-go-client/types"
 )
 
-var idGen = proxy.ReferenceIDGenerator{}
+func TestClient(t *testing.T) {
+	testCases := []struct {
+		name string
+		f    func(t *testing.T)
```

Comment:
Can we add `NoParallel` to this tests as well ?

Reply:
We can, but is it OK to do that once we need it?

Reply:
Makes sense 👍🏼